src/Ship.cpp

```diff
@@ -402,6 +402,8 @@ vector3d Ship::CalcAtmosphericForce() const
 	// Get current atmosphere parameters
 	double pressure, density;
 	GetCurrentAtmosphericState(pressure, density);
+	if (density < 1.0e-3)
```

Comment:
Is there a reason you chose `1.0e-3` instead of one of ours or the STL's float/double epsilon values? We define `is_zero_general` in `FloatComparison.h` for this sort of case so we don't have random epsilon/ULP values scattered around the code...

Reply:
**GetAtmosphericState** method of Planet gives zero values beyond atmosphere height:
```cpp
	// This model has no atmosphere beyond the adiabetic limit
	// Note: some code duplicated in InitParams(). Check if changing.
	if (dist >= m_atmosphereRadius) {
		*outDensity = 0.0;
		*outPressure = 0.0;
		return;
	}
```
then strict comparision, using is_zero_exact suitable.